napalm_logs/listener/kafka.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+'''
+Kafka listener for napalm-logs.
+'''
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+# Import stdlib
+import json
+import time
+import signal
+import logging
+
+# Import third party libs
+try:
+    import kafka
+    HAS_KAFKA = True
+except ImportError as err:
+    HAS_KAFKA = False
+
+# Import napalm-logs pkgs
+from napalm_logs.exceptions import NapalmLogsException
+from napalm_logs.listener.base import ListenerBase
+from napalm_logs.config import KAFKA_LISTENER_TOPIC
+
+log = logging.getLogger(__name__)
+
+
+class KafkaListener(ListenerBase):
+    '''
+    Kafka listener class.
+    '''
+    def __init__(self, address, port, pipe):
+        self.address = address
+        self.port = port
+        self.pipe = pipe
+        self.topic = KAFKA_LISTENER_TOPIC
+        self.__up = False
+
+    def _exit_gracefully(self, signum, _):
+        log.debug('Caught signal in listener process')
+        self.stop()
+
+    def start(self):
+        '''
+        Start listening for messages  
+        '''
+        # Start suicide polling thread
+        signal.signal(signal.SIGTERM, self._exit_gracefully)
+        self.__up = True
+        try:
+       	    self.consumer = kafka.KafkaConsumer(bootstrap_servers='{}:{}'.format(self.address, self.port), group_id='napalm-logs')
```

Comment:
This indentation doesn't look fine. Can you check?
Also, when passing multiple kwargs, please separate them on new lines.